src/components/GeoMap.js

```diff
@@ -473,7 +472,7 @@ export default class GeoMap {
         } else {
           const originalValueRaw = d.rollupValue[attrKey];
           const originalValue = (attrValue.transform) ?
-              attrValue.transform(originalValueRaw) : originalValueRaw;
+              attrValue.transform(originalValueRaw, this.maptable.rawData) : originalValueRaw;
```

Comment:
You should document it on the README.
But here you're passing the whole raw dataset as parameter, and not the raw row